dataactvalidator/config/sqlrules/b9_award_financial.sql

```diff
@@ -7,7 +7,7 @@ SELECT af.row_number,
 	af.program_activity_code
 FROM award_financial as af
 WHERE af.submission_id = {0}
-	AND CAST(COALESCE(SUBSTR(af.beginning_period_of_availa,1,4),'0') AS integer) >= 2016
```

Comment:
May have missed this during our many discussions last week, but why is the `SUBSTR` no longer necessary?